src/test/java/racing/RacingCarGameTest.java

```diff
@@ -3,40 +3,47 @@
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
 import racing.domain.strategies.CarMoveStrategy;
 import racing.domain.strategies.CarMoveStrategyImpl;
 import racing.domain.strategies.CustomRandomImpl;
 import racing.dto.GameResult;
 import racing.service.RacingCarGame;
 
+import java.util.Collections;
+import java.util.List;
+
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
 
 class RacingCarGameTest {
 
-    @DisplayName("자동차 수를 1대 보다 작게 입력하면 예외 발생")
+    @DisplayName("자동차 이름을 입력하지 않으면 예외 발생")
```

Comment:
```suggestion
    @DisplayName("자동차 이름을 입력하지 않으면 예외를 던진다. ")
```
예외의 최상위 객체인 Throwable은 던진다는 의미가 있죠. 
예외는 발생하는게 아닌 던진다는 표현이 적절합니다.